src/tree/browser/components/search-overlay.tsx

```diff
@@ -0,0 +1,88 @@
+/*********************************************************************
+ * Copyright (c) 2024 Arm Limited and others
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the MIT License as outlined in the LICENSE File
+ ********************************************************************************/
+
+import { VSCodeButton } from '@vscode/webview-ui-toolkit/react';
```

Comment:
We longer term need to look for a replacement. The toolkit has been deprecated without a successor.